datahub-web-react/src/app/entity/shared/EntityDropdown/MoveGlossaryEntityModal.tsx

```diff
@@ -46,8 +48,10 @@ function MoveGlossaryEntityModal(props: Props) {
                         duration: 2,
                     });
                     refetch();
-                    if (refetchData) {
-                        refetchData();
+                    if (isInGlossaryContext) {
+                        const oldParentToUpdate = getParentNodeToUpdate(entityData, entityType);
+                        const newParentToUpdate = selectedParentUrn || getGlossaryRootToUpdate(entityType);
+                        updateGlossarySidebar([oldParentToUpdate, newParentToUpdate], updatedUrns, setUpdatedUrns);
                     }
                 }, 2000);
```

Comment:
Ah so are we still doing the 2 second wait?

Reply:
for now unfortunately..